portals/publisher/src/main/webapp/source/src/app/data/api.js

```diff
@@ -1586,6 +1588,49 @@ class API extends Resource {
         return promised_validationResponse;
     }
 
+    static validateGraphQLByUrl(url) {
```

Comment:
Can have only one function instead of these as only the `params` are introduced in the first case

Reply:
Resolved by 49ae30a1855488741d3b997214baebcc5a2e8124